### PR TITLE
align tier 1 apis to k8s upstream GA api guidelines

### PR DESCRIPTION
**Pull request by bparees**
see:
https://github.com/kubernetes/website/pull/31389
https://kubernetes.io/docs/reference/using-api/deprecation-policy/

### CODE REVIEW

**Comment by bparees** [comment]
/assign @derekwaynecarr 
cc @deads2k @sttts

**Comment by bparees** [comment]
@openshift/team-documentation ptal

**Comment by bparees** [comment]
pushed minor typo fix, reasserting lgtm

**Comment by ahardin-rh** [comment]
@bparees @derekwaynecarr Thanks for this update! LGTM
@gangwgr Can you please provide a QE review? Thank you!

**Comment by bparees** [comment]
@ahardin-rh i don't know that there's anything for QE to review here?

File: modules/api-support-tiers.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ All commercially supported APIs, components, and features are associated under o
 [discrete]
 [id="api-tier-1_{context}"]
 == API tier 1
-APIs and application operating environments (AOEs) are stable within a major release for a minimum of 12 months or 3 minor releases from the announcement of deprecation, whichever is longer. The release that introduces a new or revised API or AOE, and the two following minor releases (n, n+1, n+2).
+APIs and application operating environments (AOEs) are stable within a major release.  They may be deprecated within a major release, but they will not be removed until a subsequent major release.
```
Comment (gangwgr) [inline modules/api-support-tiers.adoc:13]: I think extra space in this line  after "release."

Comment (gangwgr) [reply modules/api-support-tiers.adoc:13]: others are lgtm.

Comment (bparees) [reply modules/api-support-tiers.adoc]: removed, not that it matters since once rendered all spacing is unified.

**Comment by mbearer** [comment]
LGTM/approved from my perspective

**Pull request merged**